app/src/main/res/layout/activity_start.xml

```diff
@@ -30,6 +30,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginStart="@dimen/default_margin"
+            android:contentDescription="@null"
```

Comment:
There weren't `android:contentDescription` in ImageView. It's useless here, because this layer already have title and description, so I set it to `@null`